packages/node/src/__mocks__/ClientAuthentication.ts

```diff
@@ -32,3 +45,21 @@ export const mockClientAuthentication = (): ClientAuthentication =>
     LogoutHandlerMock,
     SessionInfoManagerMock
   );
+
+type CustomMocks = {
+  storage: IStorageUtility;
+  sessionInfoManager: ISessionInfoManager;
+  loginHandler: ILoginHandler;
+  redirectHandler: IRedirectHandler;
+  logoutHandler: ILogoutHandler;
+};
+
+export const mockCustomClientAuthentication = (
+  mocks: Partial<CustomMocks>
+): ClientAuthentication =>
+  new ClientAuthentication(
+    mocks.loginHandler ?? LoginHandlerMock,
+    mocks.redirectHandler ?? RedirectHandlerMock,
+    mocks.logoutHandler ?? mockLogoutHandler(mocks.storage!),
```

Comment:
A bunch of warnings here about the non-null assertion, so either explicitly initialise it or add an ESLint ignore comment. (Likewise below.)

Reply:
Weird, ESlint did not complain (that I realised), only the GH static code analysis tool takes issue with this. I'll see how to fix this.

Reply:
I think ESLint is still only set to warnings, so it's probably somewhere in your console's logs :)

Reply:
4f6ba7c2594e2dccbf474288cb6e68734c2a6d7a

Reply:
Oh yeah looking for them (in the large output of all the tests ran by lerna ^^). Should it be an error ?

Reply:
I think it might be good to turn warnings into an error, but I think there are so many of them that that's probably best done separately.